File: ssr_pkg/src/chao_node.cpp
#include <ros/ros.h>
#include <iostream>
using namespace std;
#include <std_msgs/String.h>

int main(int argc, char *argv[])
{
    ros::init(argc, argv, "chao_node");
    cout << "hello world" << endl;
    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<std_msgs::String>("kuai_shang_che_kai_hei_qun", 10);
    ros::Rate loop_rate(10);
    while (ros::ok())
    {
        cout << "I will sent mesg";
        std_msgs::String msg;
        msg.data = "machaodaifei";
        pub.publish(msg);
        cout << endl;
        loop_rate.sleep();
    }
    return 0;
}

File: ssr_pkg/scripts/chao_node.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# vim:fenc=utf-8
import rospy
from std_msgs.msg import String

if __name__ == "__main__":
    rospy.init_node("chao_node")
    rospy.logwarn("chao_node is running")
    pub = rospy.Publisher("kuai_shang_che_kai_hei_qun", String, queue_size=10)
    rate = rospy.Rate(10)
    while not rospy.is_shutdown():
        rospy.logwarn("i will sent msgs")
        msg = String()
        msg.data = "guofu machao daifei"
        pub.publish(msg)
        rate.sleep()
